src/transformers/modeling_utils.py

```diff
@@ -28,6 +28,7 @@
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 
 import torch
+from accelerate.utils.modeling import find_tied_parameters
```

Comment:
Can we just protect this by an `is_accelerate_available`? If users installed transformers and PyTorch separately, they won't have it (they'd need to do `pip install transformers["torch"]`) and in this case we just skip the test of missing tied parameters (so there would be maybe extra warning in this case).

Reply:
Thanks for the explanation !